.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.5, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
```

Comment:
The newer version (`setup-python@v2`) is used in the official document. Should we use it?

https://docs.github.com/en/actions/guides/building-and-testing-python#starting-with-the-python-workflow-template

Reply:
@matsui528 Cool! I've upgraded both setup-python and checkout actions.